float64 distance_metres     # Request message: Distance in (m) to be converted to (ft)
---                         # Demarcation
float64 distance_feet       # Response message: Distance in (ft) after Conversion
bool success                # Response message: Success or failure of conversion